readme.txt

```diff
@@ -41,6 +41,17 @@ This is compatible with WooCommerce>=2.4, including the new 3.0 release. It has
 
 == Changelog ==
 
+= 4.3.5 =
+* Fixed, multiple webhook API calls.
+* Added, subscription.charged webhook event.
+* Updated, plugin activation and deactivation hooks.
+* Electro mobile button support added.
+* Knitroot minicart and spinner issue fix.
```

Comment:
@sonyy-m Can you please remove Knitroot text from the change log?

Reply:
@ChetanGN do you want me to remove entire 49 line or only `Knitroot` ?